### PR TITLE
Enhance the documentation, add missing bits and fix outdated information

### LINKED ISSUE #2579: [dev.icinga.com #12847] Document how to add additional columns in the monitoring views
This issue has been migrated from Redmine: https://dev.icinga.com/issues/12847

**Created by elippmann on 2016-10-04 10:10:23 +00:00**

Assignee: _(none)_
Status: _New_ 
Target Version: _Backlog_
Last Update: _2016-10-04 10:10:23 +00:00 (in Redmine)_

---

---

**Relations**:

* relates [#12847](https://dev.icinga.com/issues/12847)

### LINKED ISSUE #2950: Preference documentation outdated
The preference documentation seems to be outdated. As far as i understand from @bobapple's comment in https://github.com/Icinga/puppet-icingaweb2/issues/166#issuecomment-327772662 it should be now configured by setting config_(backend|resource) in /etc/icingaweb2/config.ini

### PR DESCRIPTION
**Pull request by dnsmichi**
fixes #2579
fixes #2950

### CODE REVIEW

**Comment by lippserd** [comment]
Hi Michael,

Thanks for the huge update. Very much appreciated. Give me a go when it's ready to review.

Cheers,
Eric

**Comment by dnsmichi** [comment]
Hi Eric,

I'm still fixing the last bits. Just found out that some LDAP attributes and groups are missing. I'll rebase later on.

Kind regards,
Michael

File: doc/02-Installation.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ chapter.
 ## Installing Requirements <a id="installing-requirements"></a>
 
 * A web server, e.g. Apache or nginx
-* PHP >= 5.3.0 w/ gettext, intl, mbstring and OpenSSL support
+* PHP >= 5.3.0 with gettext, intl, mbstring and OpenSSL support
 * Default time zone configured for PHP in the php.ini file
 * LDAP PHP library when using Active Directory or LDAP for authentication
-* Icinga 2.x w/ IDO feature enabled or Icinga 1.x w/ IDO
+* Icinga 2.x with the IDO feature enabled or Icinga 1.x w/ IDO
```
Comment (nbuchwitz) [inline doc/02-Installation.md]: Line 19:
Maybe better: Icinga 2.x or 1.x with the IDO feature enabled

Comment (dnsmichi) [reply doc/02-Installation.md]: I'll refactor the chapter a bit, thanks for the pointer. I've started to remove references to 1.x in this commit in other locations already.

**Comment by dnsmichi** [comment]
Now I'm satisfied :-)

**Comment by lippserd** [comment]
Hi Michi,

Again, thanks for the PR. Looks pretty good 👍  I just changed the documentation of the theme setting because it is case-sensitive:

screen shot 2017-09-29 at 11 05 20

I'll merge it in a second.

Cheers,
Eric

**Comment by dnsmichi** [comment]
@lippserd thanks for the theme changes, did not notice that :)

**Pull request merged**